components/brex_staging/sources/new-transfer-event/new-transfer-event.mjs

```diff
@@ -0,0 +1,16 @@
+import common from "../../../brex/sources/new-transfer-event/common.mjs";
+import brexApp from "../../brex_staging.app.mjs";
+
+export default {
+  ...common,
+  dedupe: "unique",
+  type: "source",
+  key: "brex_staging-new-transfer-event",
+  name: "New Transfer Event",
```

Comment:
Since this source uses a webhook, it should have "(Instant)" at the end of the name.
```suggestion
  name: "New Transfer Event (Instant)",
```